framework/components/ecl-breadcrumbs/_breadcrumbs.scss

```diff
@@ -1,268 +1,55 @@
-/* stylelint-disable */
+/*
+ * Breadcrumbs
+ * @define breadcrumbs
+ */
+
 $breadcrumb-heigth: 2.778em;
 
-.breadcrumb {
-  position: relative;
-  margin: 0;
-  padding: 0;
-  background: none;
-  font-size: 0.833em;
-  font-weight: 600;
+.ecl-breadcrumbs {
+  @include ecl-hidden-print();
+
   border-bottom: 1px solid #fff;
-  border-radius: 0;
+  font-size: map-get($ecl-font-size, 's');
   line-height: normal;
```

Comment:
Apparently, `normal` is approximatively equal to `1.14` in modern browsers... If you remove this rule, does it really affect the appearance of the component? The less rules, the better :)